README.md

```diff
@@ -8,6 +8,7 @@ JSON Schema $Ref Parser
 [![npm](https://img.shields.io/npm/v/json-schema-ref-parser.svg)](https://www.npmjs.com/package/json-schema-ref-parser)
 [![Dependencies](https://david-dm.org/APIDevTools/json-schema-ref-parser.svg)](https://david-dm.org/APIDevTools/json-schema-ref-parser)
 [![License](https://img.shields.io/npm/l/json-schema-ref-parser.svg)](LICENSE)
+[![Buy us a tree](https://img.shields.io/badge/Treeware-%F0%9F%8C%B3-lightgreen)](https://click.treeware.earth/APIDevTools/json-schema-ref-parser)
```

Comment:
This link goes to a 404 page.  Is that just a temporary problem?  Should we instead link directly to `https://offset.earth/treeware?gift-trees` instead?

Reply:
It's best you leave the link to go via our click URL. This will allow us to track the trees in our forest which came from your project. This is a work in progress as I work with Offset.Earth guys on this. 

The URL will work once you have added the Treeware licence and then added your package to the website https://treeware.earth/#add-package-section (our website checks you have the licence in your README).

Reply:
Guys, I have added a quick message on the website to show you that the Click URL will work once you have added the README and added the package to the Treeware website.

https://click.treeware.earth/APIDevTools/json-schema-ref-parser

Hope this helps.

Reply:
That looks great @jamesmills.  Much better and more informative than the empty 404 page 👍 

Question though... what exactly do you check for to verify that the Treeware license has been added to the ReadMe?  Does it require certain wording?  Or does it just check for the badge? Or just the link?

Why even require a check at all?  Seems like an unnecessary roadblock to me that will lower the adoption rate of Treeware.  Maybe somebody doesn't want to add anything to their ReadMe, but they want to link to Treeware in their docs, or maybe even in their product.